### DISCUSSION TITLE
How to enable debug print?

### DISCUSSION THREAD

**Discussion by titan73** [The-OpenROAD-Project/OpenROAD#5939] [Q&A] [answered]
To see the output the debugPrint.

**Comment by maliberty** [comment, marked as answer]
See https://openroad.readthedocs.io/en/latest/contrib/Logger.html#debug-messages

Debug messages are enabled with the tcl command: `set_debug_level <tool> <group> <level>`

**Reply by titan73** [reply]
That works. Thanks!
